tests/suites/test_suite_pk.data

```diff
@@ -1460,52 +1460,48 @@ pk_copy_from_psa_fail:
 
 Copy from PSA: valid EC (SECP_R1_256 + ANY_HASH)
 depends_on:MBEDTLS_ECP_HAVE_SECP256R1:MBEDTLS_PK_CAN_ECDSA_SIGN:MBEDTLS_PK_CAN_ECDSA_VERIFY:MBEDTLS_MD_CAN_SHA256
-pk_copy_from_psa_success:"587CF7C57EB7C6254CBF80CC59846521B4FBCBA8BC4B362A9B043F0DEB49CCA1":PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1):256:PSA_KEY_USAGE_SIGN_HASH | PSA_KEY_USAGE_VERIFY_HASH | PSA_KEY_USAGE_EXPORT:PSA_ALG_ECDSA(PSA_ALG_ANY_HASH)
+pk_copy_from_psa_success:"587CF7C57EB7C6254CBF80CC59846521B4FBCBA8BC4B362A9B043F0DEB49CCA1":PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1):PSA_ALG_ECDSA(PSA_ALG_ANY_HASH)
 
 Copy from PSA: valid EC (SECP_R1_256 + SHA_256)
 depends_on:MBEDTLS_ECP_HAVE_SECP256R1:MBEDTLS_PK_CAN_ECDSA_SIGN:MBEDTLS_PK_CAN_ECDSA_VERIFY:MBEDTLS_MD_CAN_SHA256
-pk_copy_from_psa_success:"587CF7C57EB7C6254CBF80CC59846521B4FBCBA8BC4B362A9B043F0DEB49CCA1":PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1):256:PSA_KEY_USAGE_SIGN_HASH | PSA_KEY_USAGE_VERIFY_HASH | PSA_KEY_USAGE_EXPORT:PSA_ALG_ECDSA(PSA_ALG_SHA_256)
+pk_copy_from_psa_success:"587CF7C57EB7C6254CBF80CC59846521B4FBCBA8BC4B362A9B043F0DEB49CCA1":PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1):PSA_ALG_ECDSA(PSA_ALG_SHA_256)
```

Comment:
Yes, I think this is possible. However this goes back to the original implementation in which I used to generate the keys, which I then abandoned in favor of an explicit key in order to reduce requirements for the test.
On the pro side of your suggested change there is surely a wider support for any type of EC key, without sticking to a particular curve. I will give this a try.

Reply:
Mmm wait, I just realized that the `ONE_FAMILY` macro can also pick Montgomery keys, but as far as I know those are not allowed to do ECDSA and therefore they might require further tricks for the `pk_copy_from_psa_success` test function :/

Reply:
Oh yes, sorry, missed that. Then I think I'm more inclined to keep the current version, because generating keys comes with a computational cost - and as you say, also more dependencies (`GENPRIME` for RSA).

Reply:
> generating keys comes with a computational cost

I'd like to replace it by a hard-coded RSA key, or an RSA key that's cached between instances. But that's not going to happen before 3.6.

Reply:
Filed as https://github.com/Mbed-TLS/mbedtls/issues/8871 — let's handle that after the 3.6 code freeze.